components/ord-service/Makefile

```diff
@@ -0,0 +1,32 @@
+APP_NAME = compass-ord-service
+APP_PATH = components/ord-service
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/pr/buildpack-java:PR-3076
```

Comment:
Set proper image tag after this is merged: https://github.com/kyma-project/test-infra/pull/3076